### ISSUE TITLE
Can you provide hyperparameter settings that are not mentioned in the paper, especially those that are different in .yaml file provided

### ISSUE THREAD

**Issue by Qiuyu-Li276** [BICLab/SpikeYOLO#20]
Can you provide hyperparameter settings that are not mentioned in the paper, especially those that are different in .yaml file provided

**Comment by Qiuyu-Li276** [comment]
The difference between the learning rate and decay factor in modelEMA and the values mentioned in the paper makes me wonder if there are any other parameters that have also been modified, which are different from the configuration file currently provided

**Comment by XinhaoLuo666** [comment]
The decay factor is 0.25, which is the same as in the paper and the code. The learning rate should be set to 0.01 (we have already updated the code), and other hyperparameter settings remain the same as in the code.

**Comment by Qiuyu-Li276** [comment]
The lr0 in /ultralytics/cfg//default.yaml is still 0.004, are you sure you have already updated the code ?

**Comment by KirinZheng** [comment]
@XinhaoLuo666 I also noticed that the lr0 value in default.yaml is set to 0.004. Is there something wrong with your update? Additionally, you mentioned that the 'decay factor is 0.25,' which refers to the weight_decay value in default.yaml.
